docs/get-started/get-started-with-wordpress-com.md

```diff
@@ -0,0 +1,21 @@
+---
+post_title: Get started with Wordpress.com
+---
+
+Wordpress.com offers managed hosting, which helps optimize your website for speed, security, and performance. Managed hosting takes care of many technical aspects of your site for you. WordPress.com has many plans with increasingly powerful features for different site types and budgets.
+
+Wordpress.com provides the tools, workflows, and infrastructure you need to build and deploy your website faster. Our platform supports importing existing Wordpress installations, and it has globally distributed, secure infrastucture for serving content from data centers near the user for optimal performance.
+
+While working on a site, Wordpress.com provides tools for collaboration on your project such as staging sites, coming soon mode, and real-time backups.
+
+## Before you begin
+
+To get started, create an account on Wordpress.com.
+
+### [Sign up](https://wordpress.com/start)
+
+If you've never used Wordpress.com before, sign up for a new account
```

Comment:
They may have used it before _and_ are creating a new account, so it's best to remove assumptions. E.g. 'Sign up for a new account'. I'm not sure you need to break out multiple headings here. It could be as simple as reducing this section to:

## Before you begin
You'll need to [sign up](https://wordpress.com/start) for a new WordPress.com account or [log in](https://wordpress.com/log-in) to an existing account.